### PR TITLE
Cloud changelog - Oct 27th & Nov 10th

### PR DESCRIPTION
**Pull request by amyblais**
Cloud Oct 27th release has been paused for now, so I'm including Nov 10th Cloud release changelog in the same PR.

### CODE REVIEW

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -4,12 +4,40 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+### Improvements
+
+#### User Interface (UI)
+ - Added the ability to create a new channel along with a new board associated with the created channel.
+ - Added the ability for end users to notify their Admins to reactivate a paid plan during the delinquency period by clicking “Notify admin to update billing”.
+ - Added markdown formatting for hyperlinks when pasted into the text editor.
+ - Email notifications now also include Slack attachments.
+
+#### Administration
+ - Added a new section in the **System Console** for products. For now, it only contains Boards-specific settings.
```
Comment (amyblais) [inline source/install/cloud-changelog.md:25]: Where in the System Console is the setting located in?

Comment (agnivade) [reply source/install/cloud-changelog.md:25]: It's a new section called "products" but we don't plan to enable the FF in cloud right now. So I don't believe there is a need to mention this.

**Review by agnivade — APPROVED** [review]

**Review by esethna — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -4,12 +4,39 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
 - [Release 2022-09-08](#release-2022-09-08)
-- [Release 2022-09-01](#release-2022-09-01)
+
+## Release 2022-10-27
+
+### Improvements
+
+#### User Interface (UI)
+ - Added the ability to create a new channel along with a new board associated with the created channel.
+ - Added the ability for end users to notify their Admins to reactivate a paid plan during the delinquency period by clicking “Notify admin to update billing”.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:20]:
```suggestion
 - Added the ability for end users to notify their Admins to reactivate a paid plan during the delinquency period by selecting “Notify admin to update billing”.
```

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -4,12 +4,39 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added markdown formatting for hyperlinks when pasted into the text editor.
+ - Email notifications now also include Slack attachments.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:22]: I'm not clear on this feature as written. Mattermost email notifications include details about attachments from Slack? How so? why?

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -4,12 +4,39 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+#### Administration
+ - Added JSON-compatible nested config value parsing from environment variables.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:25]: What is the Engineering PR driving this update? It may benefit from a documentation update.

**Review by cwarnermm — COMMENTED** [review]
Feedback (and questions) provided inline

File: source/install/cloud-changelog.md (hunk 4 above)
Comment (amyblais) [reply source/install/cloud-changelog.md:25]: PR: https://github.com/mattermost/mattermost-server/pull/21049

File: source/install/cloud-changelog.md (hunk 3 above)
Comment (amyblais) [reply source/install/cloud-changelog.md]: I attempted to clarify the line. PR: https://github.com/mattermost/mattermost-server/pull/20946.

File: source/install/cloud-changelog.md (hunk 4 above)
Comment (marianunez) [reply source/install/cloud-changelog.md:25]: I think we can omit this one for cloud as it's not possible to configure plugins or env vars in the cloud environment. We can leave for the v7.5 release

**Review by marianunez — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

**Review by johndavidlugtu — APPROVED** [review]

**Review by agnivade — APPROVED** [review]

**Review by dontoisme — APPROVED** [review]

**Review by iomodo — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -4,12 +4,76 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+#### User Interface (UI)
+ - Implemented progressive image loading in the webapp.
+ - Renamed "Starter" plan to "Free" plan in product.
+ - Updated NPS plugin to version 1.3.0.
+ - Removed the user interface for the legacy "Mattermost Cloud" plan (removal effective on November 1st).
+ - When the "Custom Brand Text" is left blank with custom branding enabled, the default text is now hidden.
+
+#### Administration
+ - If an Admin encounters an invitation error “SMTP is not configured in **System Console**", a link to the **System Console** SMTP configuration is now included in the error message.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:26]:
```suggestion
 - If an Admin encounters an invitation error “SMTP is not configured in System Console", a link to the SMTP configuration within the **System Console** is now included in the error message.
```

File: source/install/cloud-changelog.md [hunk 6]
```diff
@@ -4,12 +4,76 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Crashing jobs now sets the job status to "failed".
+ - Added the ability for Cloud customers to submit their feedback for wire transfers to buy a monthly subscription.
+ - Added a new schema migration to ensure the ``ParentId`` column is dropped from the ``Posts`` table. Depending on the table size, if the column is not dropped before, a significant spike in the database CPU usage is expected on MySQL databases. Writes to the table will be limited during the migration.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:29]: Is this something that a Cloud admin would need to know or be able to take action on?

File: source/install/cloud-changelog.md [hunk 7]
```diff
@@ -4,12 +4,76 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Updated NPS plugin to version 1.3.0.
+ - Removed the user interface for the legacy "Mattermost Cloud" plan (removal effective on November 1st).
+ - When the "Custom Brand Text" is left blank with custom branding enabled, the default text is now hidden.
+
+#### Administration
+ - If an Admin encounters an invitation error “SMTP is not configured in **System Console**", a link to the **System Console** SMTP configuration is now included in the error message.
+ - Crashing jobs now sets the job status to "failed".
+ - Added the ability for Cloud customers to submit their feedback for wire transfers to buy a monthly subscription.
+ - Added a new schema migration to ensure the ``ParentId`` column is dropped from the ``Posts`` table. Depending on the table size, if the column is not dropped before, a significant spike in the database CPU usage is expected on MySQL databases. Writes to the table will be limited during the migration.  
+ - Added a logic to package product version of Boards with production builds.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:30]:
```suggestion
 - Added logic to package product version of Boards with production builds.
```

File: source/install/cloud-changelog.md [hunk 8]
```diff
@@ -4,12 +4,76 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - **System Console** settings that Cloud Admins cannot configure are now hidden.
+ - Denormalized Threads table by adding the ``TeamId`` column.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:32]: Is this relevant to Cloud admins?

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline

File: source/install/cloud-changelog.md (hunk 8 above)
Comment (amyblais) [reply source/install/cloud-changelog.md:32]: Can you confirm @iomodo?

File: source/install/cloud-changelog.md (hunk 6 above)
Comment (amyblais) [reply source/install/cloud-changelog.md:29]: Can you confirm @isacikgoz?

Comment (isacikgoz) [reply source/install/cloud-changelog.md:29]: Uh, this is not something a cloud admin can take action on. Nice catch!

File: source/install/cloud-changelog.md [hunk 9]
```diff
@@ -4,12 +4,76 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Renamed "Starter" plan to "Free" plan in product.
+ - Updated NPS plugin to version 1.3.0.
+ - Removed the user interface for the legacy "Mattermost Cloud" plan (removal effective on November 1st).
+ - When the "Custom Brand Text" is left blank with custom branding enabled, the default text is now hidden.
+
+#### Administration
+ - If an Admin encounters an invitation error “SMTP is not configured in System Console", a link to the SMTP configuration within the **System Console** is now included in the error message.
+ - Crashing jobs now sets the job status to "failed".
+ - Added the ability for Cloud customers to submit their feedback for wire transfers to buy a monthly subscription.
+ - Added a new schema migration to ensure the ``ParentId`` column is dropped from the ``Posts`` table. Depending on the table size, if the column is not dropped before, a significant spike in the database CPU usage is expected on MySQL databases. Writes to the table will be limited during the migration.
```
Comment (isacikgoz) [inline source/install/cloud-changelog.md:29]:
```suggestion
```

File: source/install/cloud-changelog.md (hunk 8 above)
Comment (esethna) [reply source/install/cloud-changelog.md:31]: No it's not. This is a backend change. 

Agree with Carrie that the stuff in the changelog should be relevant and have impact to the audience

**Review by marianunez — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]
